jni_gen/lib/src/config/yaml_reader.dart

```diff
@@ -14,19 +14,20 @@ class ConfigError extends Error {
   String toString() => message;
 }
 
-class ConfigProvider {
-  ConfigProvider.of(this.cli, this.yaml);
-  ConfigProvider.fromYaml(this.yaml) : cli = const {};
-  ConfigProvider.fromMap(this.cli) : yaml = const {};
+/// YAML Reader which allows to override specific values from command line.
```

Comment:
nit: allows -> enables

To allow is when there's rules that prohibit/disallow things.
To enable is when something was not possible before.